### ISSUE TITLE
Unable to delete files in blob after upgrade

### ISSUE THREAD

**Issue by gjadal** [microsoft/AzureStorageExplorer#2240]
**Storage Explorer Version**1.11
**Build Number**: 20191105.2
**Platform/OS**: windows 10
**Architecture**:  x64
**Regression From**: yes

## Bug Description ##
Unable to delete blob files in a blob container. the Delete is stuck at Deleting....

**Comment by JasonYeMSFT** [comment]
Can you give a screen shot of the activity log message and share the AzCopy Log with us? You can find the log file at C:\\Users\\(your_username)\\.azcopy\\(the file that matches the creation time).

**Comment by gjadal** [comment]
![image](https://user-images.githubusercontent.com/13009829/68438524-5b921080-01bc-11ea-9585-2fc3d510f734.png)
There is an empty folder called plans under the .azcopy

**Comment by gjadal** [comment]
![image](https://user-images.githubusercontent.com/13009829/68438816-508bb000-01bd-11ea-8aa8-1672f5102613.png)

**Comment by gjadal** [comment]
Can I go back to a previous version which allowed me to uncheck Azcopy integration? Even when I install a previous version, it seems to install the latest one..

**Comment by JasonYeMSFT** [comment]
You can find the previous release here. [v1.10.1 release](https://github.com/microsoft/AzureStorageExplorer/releases/tag/V1.10.1)

Based on the activity message, AzCopy hasn't finished discovery meaning that it hasn't finished listing the blob to delete. AzCopy has an aggressive retry strategy and won't show failure unless it has been retrying for a long time.

**Comment by gjadal** [comment]
How long should I wait for it to discover. Its just a single file to delete. and i waited for more than a minute. Even cancel doesnt work..

**Comment by radu87ro** [comment]
I'm experiencing similar delete action wait times. Instead of 30 sec before the upgrade to the latest version, I'm having to wait 3-5 minutes for the operation to complete.

**Comment by JasonYeMSFT** [comment]
@gjadal, we have recently fixed a bug related to **key attached Storage Accounts** where AzCopy uses incorrect endpoints. This is included in the 1.11.1 release. If it applies to your situation, you can try upgrading Storage Explorer. 

We have noticed that AzCopy is abnormally slow in some situations, but we haven't seen it being stuck. If you are still blocked by this issue, please comment below and try to find the log file for us. Or you can open another issue instead.

**Comment by JasonYeMSFT** [comment]
Is anyone still seeing this problem?
